src/thor/bidirectional_astar.cc

```diff
@@ -289,8 +291,9 @@ inline bool BidirectionalAStar::ExpandForwardInner(GraphReader& graphreader,
   }
 
   // Get end node tile (skip if tile is not found) and opposing edge Id
-  graph_tile_ptr t2 =
```

Comment:
here we can just make a copy of the tile that was passed in and use that as the cached tile in the GetGraphTile call. no need to add more tiles to the signature of ExpandForwardInner